Xamarin.Forms.Platform.Android/Forms.cs

```diff
@@ -190,17 +187,20 @@ static Color GetAccentColor()
 
 		class AndroidDeviceInfo : DeviceInfo
 		{
-			readonly IDeviceInfoProvider _formsActivity;
+			readonly Context _formsActivity;
 			readonly Size _pixelScreenSize;
 			readonly double _scalingFactor;
 
 			Orientation _previousOrientation = Orientation.Undefined;
 
-			public AndroidDeviceInfo(IDeviceInfoProvider formsActivity)
+			public AndroidDeviceInfo(Context formsActivity)
```

Comment:
I agree with @adrianknight89 on the naming; `_formsActivity` should be renamed `_context`. The bad naming is a pre-existing condition here, so I'm not going to reject the PR if it doesn't get changed, but it'd be nice.